src/Voron/Data/Tables/Table.cs

```diff
@@ -524,7 +524,7 @@ internal Tree GetTree(Slice name)
         private Tree GetTree(TableSchema.SchemaIndexDef idx)
         {
             if (idx.IsGlobal)
-                return _tx.ReadTree(idx.Name);
+                return _tx.ReadTree(idx.NameAsSlice.ToString());
```

Comment:
This requires allocating a new string on every call, which is something that happens a LOT. 
We are better off making an overload to `ReadTree` that can handle Slices.